cltk/corpus/latin/perseus_corpus_types.py

```diff
@@ -0,0 +1,314 @@
+"""`perseus_corpus_types` - a mapping of corpus types into common periods, based largely on:
+https://en.wikipedia.org/wiki/Latin_literature
+"""
+
+perseus_corpus_texts_by_type = {
```

Comment:
Something I almost asked last time, but will now. Could you put into words the different purposes between `corpus_directories_by_type` and `perseus_corpus_texts_by_type`? I take it that they allow one to collect an era's texts by different lookups … but is it anything beyond this? If not, I suggest that we either merge the two dicts or at least combine them into one module.

Reply:
Yes, I think you're right, we should provide at least one dictionary that sums the two. 

I went with two: one for individual files and one for directories, because the Latin Library corpus was organized in that fashion. And I think there's utility in keeping that distinction; e.g., some one might not agree with my type or era mapping, but at a glance they can see that the  'republican' era of the Latin Library consists of large buckets of documents in the directories: './caesar',   './lucretius',   './nepos',  './cicero'
I was thinking about expanding the directory contents into one dict of era/types and a list of files, but I didn't do that because I hoped that others reading this might see that they only might need to provide a mapping to some top level directories, if their corpus were organized in that manner.  Overall I wanted to make it as easy as possible for a corpus author to provide some generic way for external users to get reasonable access to a corpus or slices of a corpus.